action.yaml

```
@@ -23,3 +27,7 @@ runs:
     run: |
       curl -sSf https://rye-up.com/get | bash
       echo "$HOME/.rye/shims" >> $GITHUB_PATH
+  - name: setup behavior
+    shell: bash
+    if: ${{ inputs.use-uv == 'true' }}
```

FYI: `if` では `${{}}` を書かなくても式として認識されます（あってもよい）
```suggestion
    if: inputs.use-uv == 'true'
```

---

あと，https://github.com/astral-sh/rye/releases/tag/0.31.0 とか見てるとそのうち uv がデフォルトになるかもしれないですね．その時は false もあった方がよいかも？（その時はどちらかというと package installer を選ぶ input にした方がよさそうですが）